docs/core-migration-guide.md

```diff
@@ -24,6 +24,8 @@ license: |
 
 ## Upgrading from Core 3.1 to 3.2
 
+- Since Spark 3.2, fair scheduler also supports read a configuration file from the remote node. `spark.scheduler.allocation.file` can either be a local file path or HDFS file path.
```

Comment:
```suggestion
- Since Spark 3.2, the fair scheduler also supports reading a configuration file from a remote node. `spark.scheduler.allocation.file` can either be a local file path or HDFS file path.
```